examples/mobile/UIComponents/components/list/basiclist.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../lib/tau/mobile/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page">
+		<div class="ui-header" data-position="fixed">
+			<h1>
+				Basic List
+			</h1>
+		</div>
+		<div class="ui-content">
+			<ul class="ui-listview ui-content-area">
+				<li>
+					<div class="ui-li-text">
+						<span class="ui-li-text-title">
+							Main text 01
+						</span>
+						<span class="ui-li-text-sub ui-li-text-value">
+							Value text
+						</span>
+					</div>
+				</li>
+				<div class="ui-li-divider"></div>
+				<li class="ui-li-underline">
```

Comment:
I have a question.
Where is defined the "ui-li-underline"?
And what's the different "ui-li-devider" and "ui-li-underline" ?

Reply:
My mistake, I will remove the ui-li-underline.